pkg/fileservice/io_merger.go

```diff
@@ -41,7 +41,9 @@ func NewIOMerger() *IOMerger {
 
 var slowIOWaitDuration = time.Second * 10
 
-func (i *IOMerger) waitFunc(key IOMergeKey, ch chan struct{}) func() {
+var maxIOWaitDuration = time.Minute
```

Comment:
what does IOMerger merge?   I am somewhat confused.

maxIOWaitDuration is waiting for what?   Why 1 min?   It is like eternal ...